### ISSUE TITLE
Guided Tours - required fields

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#40008]
### Steps to reproduce the issue
See video recording - keyboard navigation only - all key presses highlighted on screen by external software
Creating seperate issue for each bug report so that they dont get lost inside a bigger report (again)

If a field is required you can not move forward in the tour until you complete the field ✔️
But if you then go back you have to enter something in the field again even though iy is not empty 😢
So obviously something wrong in the code to check the field has input

https://user-images.githubusercontent.com/1296369/222925048-334d6950-3170-4133-8306-36177fa98bd2.mp4

The requirement to use a mouse to successfully use the Guided Tours component is an accessibility failure and breaks our commitment to providing an accessible way to use Joomla

**Comment by brianteeman** [comment]
Addressed elsewhere
